static/app/components/events/eventStatisticalDetector/aggregateSpanDiff.tsx

```diff
@@ -112,7 +112,7 @@ function AggregateSpanDiff({event, project}: AggregateSpanDiffProps) {
     isLoading: isSpansDataLoading,
     isError: isSpansDataError,
   } = useSpanMetrics({
-    search: new MutableSearch(`transaction:${transaction} has:span.description`),
+    search: new MutableSearch(`transaction:${transaction} has:span.group`),
```

Comment:
The transaction name needs to be properly quoted and escaped in case of quotes, spaces, asterisks, etc.

The easiest way might be to do

```typescript
  const search = new MutableSearch('');
  search.addFilterValue("transaction", transaction);
```
```